veil-macros/src/lib.rs

```diff
@@ -1,87 +1,31 @@
+//! Macros for `veil`
+
+#![deny(missing_docs)]
```

Comment:
This could be a warning for development, the CI won't pass if the warnings are present due to `cargo clippy -- -D warnings`